docs/output.md

```diff
@@ -129,6 +129,18 @@ NanoPlot is used to calculate various metrics and plots about the quality and le
 
 </details>
 
+## Digital normalization with BBnorm
+
+If the pipeline is called with the `--bbnorm` option, it will normalize sequencing depth by removing reads to 1) reduce coverage of very abundant kmers and 2) delete very rare kmers (see `--bbnorm_target` and `--bbnorm_min` parameters).
+When called in conjunction with `--coassemble_group`, BBnorm will operate on interleaved (merged) FastQ files, producing only a single output file.
+The resulting FastQ files are save together with log output.
```

Comment:
Uff, I'm not sure that is a good idea, that is potentially large? Maybe make the saving optional, e.g. similar to https://nf-co.re/mag/2.3.0/parameters#save_clipped_reads and use it as in https://github.com/nf-core/mag/blob/c9468cb9154402c4a4cab78664ee50c24a91f6c3/conf/modules.config#L50

Reply:
Good that you reminded me (I should have remembered this strategy from our discussion) -- done!